Source/carthage/Build.swift

```diff
@@ -58,6 +59,7 @@ public struct BuildCommand: CommandProtocol {
 				<*> ColorOptions.evaluate(m)
 				<*> m <| Option(key: "verbose", defaultValue: false, usage: "print xcodebuild output inline")
 				<*> m <| Option(key: "project-directory", defaultValue: FileManager.default.currentDirectoryPath, usage: "the directory containing the Carthage project")
+				<*> m <| Option(key: "log-path", defaultValue: BuildCommand.defaultLogPath, usage: "path to the xcode build output. A temporary file is used by default")
```

Comment:
It'd be preferable to use a `nil` default value here. Then (1) the temporary value won't show up in the help description and (2) `openLogFile` won't need to check `path == BuildCommand.defaultLogPath`. I think you could get rid of `defaultLogPath` entirely and move the logic inside the method.